components/spreadsheet_com/package.json

```
@@ -1,16 +1,18 @@
 {
   "name": "@pipedream/spreadsheet_com",
-  "version": "0.0.1",
+  "version": "0.1.0",
   "description": "Pipedream Spreadsheet.com Components",
-  "main": "dist/app/spreadsheet_com.app.mjs",
+  "main": "spreadsheet_com.app.mjs",
   "keywords": [
     "pipedream",
     "spreadsheet_com"
   ],
-  "files": ["dist"],
   "homepage": "https://pipedream.com/apps/spreadsheet_com",
   "author": "Pipedream <support@pipedream.com> (https://pipedream.com/)",
   "publishConfig": {
     "access": "public"
+  },
+  "dependencies": {
+    "moment": "^2.29.4"
```

Need to include @pipedream/platform dependency.
```suggestion
  "dependencies": {
    "moment": "^2.29.4",
    "@pipedream/platform": "^1.5.1"
```